generators/php/codegen/src/ast/Enum.ts

```diff
@@ -48,8 +56,16 @@ export class Enum extends AstNode {
         if (this.backing != null) {
             writer.write(` : ${this.backing}`);
         }
+        if (this.serializable) {
+            writer.addReference(
+                new ClassReference({
+                    name: "JsonSerializable",
+                    namespace: ""
+                })
```

Comment:
nit: We should make this a function on the abstract context, so we can just read:
```
writer.addReference(this.context.getJsonSerializableClassReference())
```

Check out [getJsonExceptionClassReference](https://github.com/fern-api/fern/blob/6d3572ac09858e9883b96fca458463347083a4bc/generators/php/sdk/src/SdkGeneratorContext.ts#L116) for an example.

Reply:
we don't have `context` in Ast classes though right?